src/app/pages/payment/internationalPayment.component.ts

```diff
@@ -0,0 +1,153 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { InternationalPaymentService } from '../../services/payment/international-payment.service';
+import { ToastrService } from 'ngx-toastr';
+import { Location } from '@angular/common';
+import {
+  FormGroup,
+  FormBuilder,
+  Validators,
+  ReactiveFormsModule,
+  AbstractControl
+} from '@angular/forms';
+import { NgxSpinnerService } from 'ngx-spinner';
+import { Router } from '@angular/router';
+import { Subscription } from 'rxjs';
+import { SafeUrl, DomSanitizer } from '@angular/platform-browser';
+import { removeSubscription } from 'src/app/helpers/unsubscribe';
+
+@Component({
+  selector: 'app-payments',
+  templateUrl: './internationalPayment.component.html',
+  styleUrls: ['./internationalPayment.component.scss']
+})
+export class PaymentsComponent implements OnInit {
+  loading: boolean;
+  payload: any;
+  form: FormGroup;
+  formData;
+  subscription: Subscription[] = [];
+  currentUrl: SafeUrl;
+  iframeSrc: SafeUrl;
+  shouldHide = true;
+  expectedMonths: any = [
+    '01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12'
+  ];
+
+  expectedYears: any;
+
+  constructor(
+    private internationalPaymentService: InternationalPaymentService,
+    private toastrService: ToastrService,
+    private _location: Location,
+    private fb: FormBuilder,
+    private spinner: NgxSpinnerService,
+    private sanitizer: DomSanitizer
+
+  ) { }
+
+  generateYears() {
+    const thisYear = new Date().getFullYear();
+    this.expectedYears = Array.apply(0, Array(14))
```

Comment:
Hey @patrickf949 ,great job working on the feature, I may have an alternative for this method. We can use `Array.from(Array(14).keys())` because it generates an array of numbers and as a result, we do not need to `_element` and `index` properties in the map function, we could just have 1 property. This can also work in your favor since tslint is compaining that `_element` should be camelCased
Here's a screenshot demonstrating the functionality that I'm proposing and how you can use it 

![image](https://user-images.githubusercontent.com/12128153/63406996-20b9e100-c3f4-11e9-8b95-cf6aebd48fad.png)

Reply:
> Hey @patrickf949 ,great job working on the feature, I may have an alternative for this method. We can use `Array.from(Array(14).keys())` because it generates an array of numbers and as a result, we do not need to have `_element` and `index` properties in the map function, we could just have 1 property. This can also work in your favor since tslint is compaining that `_element` should be camelCased
> Here's a screenshot demonstrating the functionality that I'm proposing and how you can use it
> 
> ![image](https://user-images.githubusercontent.com/12128153/63406996-20b9e100-c3f4-11e9-8b95-cf6aebd48fad.png)

Thanks @Tevinthuku done